docs/virtual_machines/host-devices.md

```diff
@@ -56,7 +56,11 @@ echo 0000:65:00.0 > /sys/bus/pci/drivers/vfio-pci/bind
 
 ## Preparation of mediated devices such as vGPU
 
-At this time, configuration of a Mediated device (mdev) should be done according to the vendor directions. Once the mdev is configured, KubeVirt will be able to discover and use it for device assignment.
+In general, configuration of a Mediated devices (mdevs), such as vGPUs, should be done according to the vendor directions. 
+KubeVirt can now facilitate the creation of the mediated devices / vGPUs on the cluster nodes. This assumes that the required vendor driver is already installed on the nodes.
+See the [Automatic configuration of mediated devices / vGPUs](</#operations/mediated_devices_configuration.md>) to learn more about this functionality.
```

Comment:
On the preview, that link takes me to `virtual_machines/host-devices/!--#operations/mediated_devices_configuration.md--`, weird